### PR TITLE
Local Environment Configuration

### PR DESCRIPTION
**Pull request by tarecord**
This sets up a local environment, symlinking the repository root into the local environment and providing a basic configuration so that the library can be worked on in a simplified environment.

Before, the environment had to be cloned within the telemetry starter plugin and trying to use xdebug was a mess.

### CODE REVIEW

**Review by ChrisMKindred — COMMENTED** [review]
This is great, I have a couple of comments but this should really help speed up dev.

File: dev/lando/.env
[the remark this answers is not in the document]
Comment (tarecord) [reply dev/lando/.env:1]: Yep, just moving too fast.

File: dev/public/index.php
[the remark this answers is not in the document]
Comment (tarecord) [reply dev/public/index.php:1]: Normally, yes. But this specifically requires the blog header from the `wp` directory.

File: .lando.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+env_file:
+  - dev/lando/.env
+
+excludes:
+  - vendor
+  - .vscode
+  - .github
+
+config:
+  php: '8.1'
```
[the remark this answers is not in the document]
Comment (tarecord) [reply .lando.yml:12]: I'll reduce this to 7.4 as that's the majority of the reporting sites (52%). ~2% of reporting sites are showing less than 7.4

File: dev/lando/install-wp.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  wp core download \
+    --version=${WP_VERSION:-latest} \
+    --force
+}
+
+function install_wp() {
+  echo "* Installing WordPress..."
+
+  # Wait for the database service to actually be ready...
+  sleep 5
```
[the remark this answers is not in the document]
Comment (tarecord) [reply dev/lando/install-wp.sh:26]: It does, but all systems default to the unit of time as seconds. I don't think we need to worry too much about it.

**Pull request merged**